### PR TITLE
Issue #61: Fix GitHub link in footer as suggested

### LINKED ISSUE #61: Footer link GH points to https://github.com/coopdevs
Should be https://github.com/openfoodfoundation/openfoodnetwork

### PR DESCRIPTION
**Pull request by lumie31**
#### What does this PR do?
- Fixes the GitHub link in page footer as suggested

closes #61 

#### How should this be manually tested?
- This can be tested by clicking on the link and following the URL

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ node_modules/
 .DS_Store
 _site
 .sass-cache/
+.jekyll-metadata
```
Comment (sauloperez) [inline .gitignore]: :ok_hand:

File: docker-compose.yml [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
+version: "2"
 services:
   web:
     build: .
     volumes:
-     - .:/app
+      - .:/app
     ports:
-     - "4000:4000"
+      - "4000:4000"
```
Comment (sauloperez) [inline docker-compose.yml]: Thanks!

**Comment by sauloperez** [comment]
Closes #61

File: .gitignore (hunk 1 above)
Comment (lumie31) [reply .gitignore]: So glad I could contribute... I liked that I was able to see a use-case for docker and thanks for helping out too.

**Comment by lumie31** [comment]
The re-review button was clicked in error, my bad

File: .gitignore (hunk 1 above)
Comment (sauloperez) [reply .gitignore]: and I'm glad that asking contributors to run docker wasn't more complex to setup Ruby as I feared.

**Pull request merged**